src/content/docs/en/pages/deploy-journey/provision-files/provision-file.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Provision files
+description: aaaa
+meta_tags: deploy, orchestrator
+namespace: docs_deploy_install_agent
+menu_namespace: deployMenu
+permalink: /documentation/products/guides/deploy/provision-files/
+---
+
+To be able to orchestrate services on your device, you need to configure all the resources. These resources can be: 
+
+- **Text**: for content being copied as plain text to the edge node.
+- **Shell Script**: for resources that will be installed and run according to the selected *trigger*
+
+You can create a resource using:
+
+- [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/)
+- [Azion API](https://api.azion.com/)
+
+---
+
+## Via RTM
+
+1. Go to [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Libraries** > **Edge Services**.
+2. Select the edge service you want to add a resource.
+3. Go to the **Resources** tab.
+5. Click on **Add Resource**.
+6. Enter the filepath of the resource.
+7. Choose the type **Text**.
+8. Add the following content to the content block: 
+
+```bash
+My name is {{name}}, and this is my first added resource
+```
+
+:::note 
+The tag {{name}} will be replaced by the environment configured for the service or the variable configured on the node. The edge node variable will override the variable configure on the edge service
+:::
+
+import Button from '~/components/Button.astro'
+
+<Button href="" text="go to work with variables" variant="secondary" target="_blank"></Button>
+
+---
+
+## Via API
+
+1. Run the following `LIST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and retrieve the edge service ID:
+
+```bash 
+curl --location 'https://api.azionapi.net/edge_services/' \
+--header 'Accept: application/json; version=3' \
+--header 'Authorization: Token [TOKEN VALUE]'
+```
+
+2. Run the following `POST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/), `:id` with the ID of the edge service and informing the following parameters in the request body:
+
+Request body properties: 
+
+| Property | Description | Required | 
+| - | - | - | 
+| `content_type` | Content type of the resource being created | Yes |
+| `name` | Name of the resource being created | Yes |
+| `content` | Content that defines actions performed when the resource state changes in the edge node | Yes |
+
+
+```bash 
+curl --location 'https://api.azionapi.net/edge_services/:id/resources' \
+--header 'Accept: application/json; version=3' \
+--header 'Authorization: Token [TOKEN VALUE]' \
+--header 'Content-Type: application/json' \
+--data '{
+  "content_type": "Text",
+  "name": "/resource",
+  "content": "My name is {{name}}, and this is my first added resource"
+}'
+```
+
```

Comment:
```suggestion
```bash 
curl --location 'https://api.azionapi.net/edge_services/:id/resources' \
--header 'Accept: application/json; version=3' \
--header 'Authorization: Token [TOKEN VALUE]' \
--header 'Content-Type: application/json' \
--data '{
  "content_type": "Text",
  "name": "/resource",
  "content": "My name is {{name}}, and this is my first added resource"
}'
```

Request body properties: 

| Property | Description | Required | 
| - | - | - | 
| `content_type` | Content type of the resource being created | Yes |
| `name` | Name of the resource being created | Yes |
| `content` | Content that defines actions performed when the resource state changes in the edge node | Yes |
```